openvdb/openvdb/tree/RootNode.h

```diff
@@ -2786,6 +2805,15 @@ RootNode<ChildT>::probeNode(const Coord& xyz)
 }
 
 
+template<typename ChildT>
+template<typename NodeT>
+inline const NodeT*
+RootNode<ChildT>::probeNode(const Coord& xyz) const
```

Comment:
this implementation is so compact it should be inlined in the deceleration. It also improves the readability